src/main/java/com/akademia/educators_back/service/ProblemInterface.java

```diff
@@ -0,0 +1,14 @@
+package com.akademia.educators_back.service;
+
+import com.akademia.educators_back.to.ProblemTo;
+
+import java.util.List;
+
+public interface ProblemInterface {
```

Comment:
Zamiast przyrostka Interface nie dawałbym nic, a przynajmniej nie to słowo. Jak mamy jakieś interfejsy w Javie to też nie nie mają tego przyrostka np: List